.env.example

```
@@ -1,11 +1,21 @@
 
 # App Configuration
 VITE_APP_URL=http://localhost:5173
 
-# Google Tag Manager
-# Get your GTM container ID from https://tagmanager.google.com
-# Format: GTM-XXXXXX
-VITE_GTM_ID=GTM-XXXXXX
+# PostgreSQL Database Configuration (Required for admin panel)
+DB_HOST=localhost
+DB_PORT=5432
+DB_NAME=your_database_name
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_NAME key should go before the DB_PORT key

(UnorderedKey)